src/ONGR/DemoMagentoBundle/Controller/CustomerController.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\DemoMagentoBundle\Controller;
+
+use ONGR\DemoMagentoBundle\Magento\Cart;
+use ONGR\DemoMagentoBundle\Magento\Customer;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Templating\EngineInterface;
+
+/**
+ * Class UserController.
+ */
+class CustomerController
+{
+    /**
+     * @var Customer
+     */
+    private $customer;
+
+    /**
+     * @var Cart
+     */
+    private $cart;
+
+    /**
+     * @var EngineInterface
+     */
+    private $templating;
```

Comment:
I am not entirely sure about this variable name. Maybe `templatingEngine`? or `templateEngine`?